### PR TITLE
305 lumen shares

### PR DESCRIPTION
**Pull request by kardan**
Removed the db parts since we are using an external service for now. But kept the files since we might not always do it (at least not for local testing).

### CODE REVIEW

File: docker/lumen/db/provision.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" <<-EOSQL
+    CREATE ROLE lumen WITH PASSWORD 'password' CREATEDB LOGIN;
```
Comment (iperdomo) [inline docker/lumen/db/provision.sh]: `'password'` should be a param to the script?

File: docker/lumen/db/provision.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    CREATE DATABASE lumen
+    WITH OWNER lumen
+      TEMPLATE = template0
+      ENCODING = 'UTF8'
+      LC_COLLATE = 'en_US.UTF-8'
+      LC_CTYPE = 'en_US.UTF-8';
+EOSQL
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "lumen" <<-EOSQL
+    CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
```
Comment (iperdomo) [inline docker/lumen/db/provision.sh]: There is no need to create the `plpgsql` extension as is enabled by default?

File: docker/lumen/db/provision.sh (hunk 1 above)
Comment (kardan) [inline docker/lumen/db/provision.sh]: Yes, this was just a proof of concept. It's not used.

File: docker/lumen/db/provision.sh [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    WITH OWNER = lumen
+    TEMPLATE = template0
+    ENCODING = 'UTF8'
+    LC_COLLATE = 'en_US.UTF-8'
+    LC_CTYPE = 'en_US.UTF-8';
+EOSQL
+
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "lumen_tenant_1" <<-EOSQL
+    CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
```
Comment (iperdomo) [inline docker/lumen/db/provision.sh]: Idem :)

File: docker/lumen/db/provision.sh (hunk 1 above)
Comment (iperdomo) [inline docker/lumen/db/provision.sh]: Then let's remove the script :)

**Pull request merged**